src/main/java/org/folio/innreach/client/InventoryClient.java

```diff
@@ -12,4 +13,7 @@
 
   @GetMapping("/inventory/instances/{id}")
   InventoryInstanceDTO getInstanceById(@PathVariable("id") UUID instanceId);
+
+  @PostMapping("/inventory/start-initial-contribution")
```

Comment:
the endpoint will be `/instance-storage/instances/iteration` and it accepts body with topic name